packages/mdc-list/_mixins.scss

```diff
@@ -393,6 +391,30 @@
   }
 }
 
+@mixin mdc-list-item-disabled-text-opacity($opacity, $query: mdc-feature-all()) {
+  $feat-structure: mdc-feature-create-target($query, structure);
```

Comment:
This mixin can take disabled color with rgba value instead. We want to move away from controlling opacity.

Reply:
I think this feature target is color for this.